apps/web-giddh/src/app/ledger/components/newLedgerEntryPanel/newLedgerEntryPanel.component.ts

```diff
@@ -1022,11 +1012,19 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
         this.calculateTotal();
     }
 
-    public calculateConversionRate(baseModel) {
+    /**
+     * Calculates conversion rate
+     *
+     * @param {*} baseModel Value to be converted
+     * @param {number} [customDecimalPaces] Optional custom decimal places (required for Rate as 4 digits are required for rate)
+     * @returns Converted rate
+     * @memberof NewLedgerEntryPanelComponent
+     */
+    public calculateConversionRate(baseModel: any, customDecimalPaces?: number): number {
```

Comment:
Correct spelling of Places in customDecimalPlaces

Reply:
Done